<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ui_newsms</class>
  <widget class="QWidget" name="ui_newsms" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>610</width>
        <height>461</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item row="4" column="0" >
        <widget class="KTextEdit" name="smsText" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>3</verstretch>
            </sizepolicy>
          </property>
        </widget>
      </item>
      <item row="5" column="0" >
        <widget class="QFrame" name="statusBarFrame" >
          <property name="frameShape" >
            <enum>NoFrame</enum>
          </property>
          <property name="frameShadow" >
            <enum>Raised</enum>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="K3ListView" name="lv_numbers" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>7</vsizetype>
                  <horstretch>1</horstretch>
                  <verstretch>2</verstretch>
                </sizepolicy>
              </property>
              <column>
                <property name="text" >
                  <string>Number</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text" >
                  <string>Name</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
            </widget>
          </item>
          <item>
            <layout class="QVBoxLayout" >
              <item>
                <layout class="QHBoxLayout" >
                  <item>
                    <widget class="KLineEdit" name="editNumber" />
                  </item>
                  <item>
                    <widget class="KPushButton" name="pickNumber" >
                      <property name="text" >
                        <string>&amp;Browse</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <widget class="KPushButton" name="buttonAdd" >
                  <property name="enabled" >
                    <bool>false</bool>
                  </property>
                  <property name="text" >
                    <string>&amp;Add</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KPushButton" name="buttonRemove" >
                  <property name="enabled" >
                    <bool>false</bool>
                  </property>
                  <property name="text" >
                    <string>Remo&amp;ve</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer1" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>61</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="textLabel2" >
          <property name="text" >
            <string>Destination Numbers</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>Message Text</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >ktextedit.h</include>
    <include location="local" >k3listview.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
</ui>
